src/components/text-editor/examples/text-editor-with-inline-images-file-storage.tsx

```
@@ -0,0 +1,116 @@
+ * `insertImage` method.
+ *
+ * Is the image upload fails, you can insert a failed thumbnail using the
+ * `insertFailedThumbnail` method.
+ */
+@Component({
+    tag: 'limel-example-text-editor-with-inline-images-file-storage',
+    shadow: true,
+})
+export class TextEditorWithInlineImagesExample {
+    @State()
+    private value: string = 'Copy an image file and paste it here.';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)